src/components/layers/download/styles/DataDownloadDialogActions.module.css

```diff
@@ -0,0 +1,5 @@
+.btnProgress {
+    position: absolute;
+    top: -10px;
```

Comment:
One could argue that this isn't a scalable way position the circular loader in the center of the button, since if the button dimensions change (in `ui`) this will be incorrectly positioned. But I think the button changing dimensions is unlikely so 🤷‍♂️

Reply:
Good point. I've changed to `margin: auto` so it will always center regardless of the button size. 

<img width="222" alt="Screenshot 2020-10-20 at 11 21 14" src="https://user-images.githubusercontent.com/548708/96567056-9fdfc900-12c6-11eb-8634-8116288412e5.png">